src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import './App.css';
+import Splash from './components/userAccess/splash';
+import Navigation from './components/Nav/navigation';
+import Doctors from './components/pages/doctors';
+import NewReservation from './components/pages/newReservation';
+import MyReservations from './components/pages/myReservations';
+import AddDoctor from './components/pages/addDoctor';
+import DeleteDoctor from './components/pages/deleteDoctor';
+import DoctorDetails from './components/doctor/doctorDetails';
+import Layout from './components/Nav/layout';
+import { useUser } from './components/userAccess/userContext';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪

Reply:
Thank you